client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -285,7 +285,9 @@ export const handlePlatformCheckoutEmailInput = ( field, api ) => {
 				wcpayTracks.recordUserEvent(
 					wcpayTracks.events.PLATFORM_CHECKOUT_OTP_COMPLETE
 				);
-				api.initPlatformCheckout().then( ( response ) => {
+				api.initPlatformCheckout(
+					e.data.platformCheckoutUserSession
```

Comment:
Since this value is delivered over `window.postMessage()`, I am concerned that it can be captured by the merchant site and abused. Is it possible to transmit the `user_session` from the platform checkout site to the merchant site without the merchant site being aware of what it is?

Reply:
The concern is valid and I believe we have it sorted by making the session a one-time use token – it has been addressed in the platform checkout end. I've replied your question in the platform checkout's PR, can we continue this discussion there?